src/entryPoints/AuthModule/UserInfo/UserInfoProvider.res

```diff
@@ -10,13 +10,28 @@ let make = (~children) => {
   let (userInfo, setUserInfo) = React.useState(_ => UserInfoUtils.defaultValueOfUserInfo)
   let fetchApi = AuthHooks.useApiFetcher()
   let {xFeatureRoute} = HyperswitchAtom.featureFlagAtom->Recoil.useRecoilValueFromAtom
+  let userOptionalRef = Some(React.useRef(userInfo))
+
+  let updateOptionalUserInfoRef = updatedUserInfo => {
```

Comment:
change the function name to updateUserInfoRef

Reply:
updated